lib/lost-utility.js

```diff
@@ -1,9 +1,96 @@
 var newBlock = require('./new-block.js');
 
+/**
+ * Returns a sanitized three-number array representing RGB color values, with a safe default.
+ * @param string
+ * @returns {number[]}
+ */
 function getColorValue(string) {
-  var color = string.split('rgb(')[1];
-  color = color.split(')')[0];
-  return color;
+  if (testRgb.test(string)) {
+    return safeRgbToRgb(string);
+  }
+  if (testHex.test(string)) {
+    return safeHexToRgb(string);
+  }
+  return [0, 0, 255];
+}
+
+var testRgb = new RegExp(/rgb/);
```

Comment:
It sounds silly, but could this either be at the top or the bottom of the file? In the middle for a "const" feels a bit odd.

Reply:
I updated organization and added tests, it should be green for both tests & lints now.
Thanks for your guidance.